app/models/miq_group.rb

```diff
@@ -67,17 +68,8 @@ def self.allows?(_group, _options = {})
     true # Remove once this is implemented
   end
 
-  def self.add(attrs)
-    group = new(attrs)
-    if group.resource.nil?
-      groups = where(:resource_id => nil, :resource_type => nil).order("sequence DESC")
-    else
-      groups = group.resource.miq_groups.order("sequence DESC")
-    end
-    group.sequence = groups.first.nil? ? 1 : groups.first.sequence + 1
-    group.save!
-
-    group
```

Comment:
Discussed with @gtanzillo and @kbrock .  This is really `create` now that the sequence is being set through `default_value_for`.  The UI does not call this method and we could not identify any other callers.